mobile/lib/features/wallet/wallet_history_item.dart

```diff
@@ -98,10 +98,13 @@ class WalletHistoryItem extends StatelessWidget {
       }
     }();
 
-    var amountFormatter = NumberFormat.compact(locale: "en_IN");
+    var amountFormatter = NumberFormat.compact(locale: "en_UK");
```

Comment:
No, this is a small chore fix which I could extract. It's just weird that we use `en_IN`, and since Indian English has got lakh and other ways to abbreviate numbers I thought it could be slightly dangerous too